.env.example

```
@@ -1,14 +1,26 @@
+#
+# NEVER commit .env — it contains secrets and is gitignored.
+
+NEXTJS_ENV
+
 # Resend API key for email sending
 NEXT_PUBLIC_resend
 # Email address for contact form submissions
 NEXT_PUBLIC_dataEmail
-# Cloudflare Turnstile — bot protection for contact form
+
+NEXT_PUBLIC_VERCEL_URL
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_VERCEL_URL key should be with a value or have an equal sign

(KeyWithoutValue)